ddtrace/appsec/_asm_request_context.py

```diff
@@ -120,7 +120,7 @@ def __init__(self):
 
     def finalise(self):
         if self.active:
-            env = core.get_item("asm_env")
+            env = self.execution_context.get_item("asm_env")
             # assert _CONTEXT_ID.get() == self._id
```

Comment:
hmmmm I know it's not a code change you made, but should this be in here?

Reply:
This change is intentional. It ensures that the `asm_env` variable is grabbed from the intended `ExecutionContext` instance rather than whichever one happens to be "active" according to `internal.core`.

Reply:
Oh... are we talking about the same line? 

            # assert _CONTEXT_ID.get() == self._id

Reply:
Oh, I see. Yeah I don't know what that's about. It should go away, but I didn't want to include that change here.